lib/src/metta/runner/modules/mod_names.rs

```diff
@@ -48,6 +50,29 @@ pub(crate) fn mod_name_remove_prefix<'a>(name: &'a str, prefix: &str) -> Option<
     }
 }
 
+//TODO-NEXT: This function is likely unneeded, and can be removed
+// /// Returns the portion of `name` that does not match the beginning of `other`, starting
+// /// at the point where the strings diverge.  For example, is `name == "top:mod_a:hello"`
+// /// and `other == "top:mod_a:goodbye"`, then this function would return "hello".
+// pub(crate) fn remove_common_prefix<'a>(name: &'a str, other: &str) -> &'a str {
+//     let mut start = 0;
+//     let mut other_iter = other.chars();
+//     for (idx, c_name) in name.char_indices() {
+//         if c_name == MOD_NAME_SEPARATOR {
+//             start = idx+1;
+//         }
+//         match other_iter.next() {
+//             Some(c_other) => {
+//                 if c_name != c_other {
+//                     return &name[start..]
+//                 }
+//             },
+//             None => return &name[start..]
+//         }
+//     }
+//     ""
+// }
+
```

Comment:
Fair point...  I didn't delete it initially because I went back and forth several times on whether it was needed.  But it's not very hard to reimplement.

Reply:
And anyway it will be in a history. We can leave it as it if you are going to cleanup it after finishing work on imports.